Multiplication.cs

```diff
@@ -1,18 +1,11 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
```

Comment:
Have you already tested that it is still woking?

Reply:
Yes, I did